site/en/docs/privacy-sandbox/attribution-reporting/index.md

```diff
@@ -243,21 +244,43 @@ Most notably, with the Attribution Reporting API proposed by Chrome:
 ## What browser configuration is available? {: #browser-configuration }
 
 - Users can opt out of the API via the user settings at `chrome://settings/privacySandbox`.
-- The API is not active in **Incognito** mode. 
-- The API is not active when **third-party cookies** are disabled. 
+- The API is not active in **Incognito** mode.
+- The API is not active when **third-party cookies** are disabled.
 
 {% Aside %}
 
 The API does not rely on third-party cookies. However, in the testing phase, third-party cookies need to be enabled for the API to be enabled. This allows developers to get debug reports and compare the API's results with cookie-based attribution.
 
 {% endAside %}
 
-
 ## How can sites control access? {: #sites-control }
 
-Arbitrary third-parties can't use the API without a publisher's or advertiser's knowledge. The
-  Attribution Reporting API needs to be enabled in child iframes via a [Permissions
-  policy](https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy).
+If the API is available in a given browser, it's available by default in any given site, both in top-level documents and scripts, and in same-origin iframes.
+
+Arbitrary third-parties—for example, cross-origin ad iframes that were not added to the page via a script that has top-level access—can't use the API without a publisher's or advertiser's knowledge: in these iframes, the Attribution Reporting API needs to be explicily enabled via a [Permissions
+policy](https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy).
+
+```html
+<iframe src="..." allow="attribution-reporting"></iframe>
+```
+
+Third parties with top-level access that add cross-origin iframes to a page can enable the Attribution Reporting API via this [Permissions
+policy](https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy) as well.
+
+{% Aside %}
+
+**What's the security benefit?**
+
+By doing this, a script with top-level access allows the frames it adds itself to use Attribution Reporting. This isn't an issue, because only a third-party script that is trusted by the site should be given top-level access anyway.
+The main security advantage of the policy lies somewhere else: frames that were not added via a top-level script should not be trusted by default to register sources or triggers, hence the requirement for the top-level site to explicitly enable the API for these.
```

Comment:
```suggestion
With Permissions Policy, a script with top-level access can allow the frames it adds
to use Attribution Reporting. This isn't a security issue, because only a third-party
script that is trusted by the site should be given top-level access.
The main security advantage of the policy lies somewhere else: frames that were not added via a top-level script should not be trusted by default to register sources or triggers, hence the requirement for the top-level site to explicitly enable the API for these.
```